metadata/src/test/java/org/apache/kafka/metadata/PartitionRegistrationTest.java

```diff
@@ -371,7 +371,7 @@ public void testPartitionRegistrationToRecord_ElrShouldBeNullIfEmpty() {
             setPartitionEpoch(0);
         List<UnwritableMetadataException> exceptions = new ArrayList<>();
         ImageWriterOptions options = new ImageWriterOptions.Builder().
-            setMetadataVersion(MetadataVersion.IBP_3_8_IV0).
+            setMetadataVersion(MetadataVersion.IBP_3_9_IV0).
```

Comment:
We have the following code in this class. Should we change IBP_3_8_IV0 to IBP_3_9_IV0?

```
    private static Stream<Arguments> metadataVersionsForTestPartitionRegistration() {
        return Stream.of(
            MetadataVersion.IBP_3_7_IV1,
            MetadataVersion.IBP_3_7_IV2,
            MetadataVersion.IBP_3_8_IV0
        ).map(mv -> Arguments.of(mv));
    }

```

Reply:
good find, fixed